cli/lib/kontena/plugin_manager.rb

```diff
@@ -174,10 +174,9 @@ def load_plugins
                 $stderr.puts " [#{Kontena.pastel.red('error')}] Plugin #{Kontena.pastel.cyan(plugin_name)} (#{spec.version}) is not compatible with the current cli version."
                 $stderr.puts "         To update the plugin, run 'kontena plugin install #{plugin_name}'"
               end
-            rescue LoadError => ex
-              $stderr.puts " [#{Kontena.pastel.red('error')}] Failed to load plugin: #{spec.name}"
+            rescue Exception => ex
```

Comment:
>> Why don't we just rescue LoadError?
> That alone won't catch syntax errors or any runtime errors or whatever that may occur during plugin loading.

Looking at the ruby [`Exception` docs](https://ruby-doc.org/core-2.3.1/Exception.html), there's a `ScriptError` that's a superclass of `LoadError` and `SyntaxError`. Would catching `ScriptError` + `StandardError` be enough to skip broken plugins?

>> That sounds like a bug to me.
> If a plugin was loading forever that would be the only way to reach kontena plugin uninstall.

Could we avoid this if the `kontena plugin` commands didn't actually load the plugins, meaning that they could still be uninstalled even if pathologically broken like this?